lib/tsconfig/3.8.x/tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowUmdGlobalAccess": false,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "alwaysStrict": true,
+    "charset": "utf8",
+    "declaration": true,
+    "diagnostics": false,
+    "downlevelIteration": true,
+    "emitBOM": false,
+    "emitDecoratorMetadata": true,
+    "emitDeclarationOnly": false,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": false,
+    "importHelpers": true,
+    "incremental": true,
+    "inlineSourceMap": false,
+    "inlineSources": true,
+    "isolatedModules": false,
+    "listEmittedFiles": false,
+    "listFiles": false,
+    "module": "es2020",
```

Comment:
Why is this done already in this config for TS 3.8? As far as I can see, this is suggested as from Angular 10: see [Why "es2020" instead of "esnext"?](https://angular.io/guide/migration-update-module-and-target-compiler-options#why-es2020-instead-of-esnext)